### ISSUE TITLE
help links are broken

### ISSUE THREAD

**Issue by DioEgizio** [PolyMC/PolyMC#377]
### Operating System

Windows, macOS, Linux, Other

### Description of bug

Help links redirect to a 404 page

### Steps to reproduce

Click any help link and see

### Suspected cause

MultiMC relics

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by glowiak** [comment]
Works fine for me. Which version are you using?

**Comment by DioEgizio** [comment]
What how.
Click the "help" button and see

**Comment by Scrumplex** [comment]
Yeah most help pages don't exist (yet)

**Comment by DioEgizio** [comment]
> Yeah most help pages don't exist (yet)

actually, help pages exist but links are different

**Comment by DioEgizio** [comment]
for example, clicking "Help" on Java Settings directs to https://polymc.org/wiki/Java-settings, but https://polymc.org/wiki/getting-started/installing-java/ exists

**Comment by Scrumplex** [comment]
Nah, not really. The help pages should explain the UI. So https://polymc.org/wiki/Java-settings should tell you what all the settings mean.

**Comment by DioEgizio** [comment]
oh

**Comment by DioEgizio** [comment]
but another example, clicking on mod/modpack help direct to https://polymc.org/wiki/(the name of the platform)-platform, but https://polymc.org/wiki/getting-started/download-mods/ and https://polymc.org/wiki/getting-started/download-modpacks/ exist

**Comment by Scrumplex** [comment]
But the pages we already have just tell you *where* you can download mods/modpack, not really how the UI works

**Comment by DioEgizio** [comment]
oh

**Comment by ZekeZDev** [comment]
Would it be worth adding a second wiki for this on the polymc website that has all of the infomation from multimc's wiki. This way we can have the help links work until we recreate them all for polymc.

**Comment by DioEgizio** [comment]
A lot of parts of the multimc wiki are on the polymc ones, except for stuff like this

**Comment by DioEgizio** [comment]
I've done the java settings help page at https://github.com/PolyMC/polymc.github.io/pull/89, I'll try to do as many help pages as possibile in my limited time in the next days(/weeks), if someone can help then plz contribute
